<?xml version="1.0" ?>
<launch>
	<node pkg="irobot" type="teleop_joy" name="teleop_joy" />
	<param name="axis_linear" value="1" type="int" />
	<param name="axis_angular" value="0" type="int" />
	<node respawn="true" pkg="joy" type="joy_node" name="joy_node">
	<param name="dev" type="string" value="/dev/input/js0" />
    	<param name="deadzone" value="0.12" />
	</node>
</launch>
